content/projects/data-science-specific/cross-validation-and-simple-linear-regression/_index.md

```diff
@@ -35,7 +35,7 @@ title: Cross-validation & Simple Linear Regression
 - Cross-validation
 - Root Mean Square Error
 
-## Background materials
+## Pre-requisites
```

Comment:
I think this could be a bit confusing since we have a section at the top that lists projects and topics that are hard and soft pre-requisites. If it should be a pre-requisite then we should possibly create a new topic that houses these materials?